backend/src/main/kotlin/app/ehrenamtskarte/backend/import/stores/bayern/steps/MapFromLbe.kt

```diff
@@ -22,6 +23,10 @@ class MapFromLbe(
     override fun execute(input: List<LbeAcceptingStore>) =
         input.mapNotNull {
             try {
+                val discounts = it.discount.clean(false)?.let { discount ->
+                    mapOf(LanguageCode.DE to discount)
+                } ?: emptyMap()
```

Comment:
🙃 
```suggestion
                val discounts = it.discount.clean(false)?.let { mapOf(LanguageCode.DE to it) }
                    ?: emptyMap()
```

Also I would not mind keeping this inlined, but matter of taste.